test/check-machines-nics

```diff
@@ -139,6 +140,27 @@ class TestMachinesNICs(VirtualMachinesCase):
         b.wait_not_present(".pf-c-modal-box")
         b.wait_not_present("#vm-subVmTest1-network-1-mac")
 
+        # Detach NIC when the VM is shutoff, Also check this issue: https://bugzilla.redhat.com/show_bug.cgi?id=1791543
+        self.performAction("subVmTest1", "off")
```

Comment:
Does this work? I think you need forceOff

Reply:
It works, since there is a wait(lambda: "login as 'cirros' user." in self.machine.execute("cat {0}".format(args["logfile"])), delay=3) in previous lines